src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/DockerProcessor.kt

```diff
@@ -17,27 +17,32 @@ package com.bardsoftware.backend.fleet.rmarkdown
 
 import com.spotify.docker.client.DefaultDockerClient
 import com.spotify.docker.client.DockerClient
+import com.spotify.docker.client.DockerClient.LogsParam.stderr
+import com.spotify.docker.client.DockerClient.LogsParam.stdout
 import com.spotify.docker.client.messages.ContainerConfig
 
 class DockerProcessor {
     private val docker: DockerClient = DefaultDockerClient.fromEnv().build()
 
-    fun getMd5Sum(message: String): String? {
+    fun getMd5Sum(message: String): String {
         var containerId: String? = null
 
         try {
             val quotedMessage = "\"$message\""
 
             val containerConfig = ContainerConfig.builder()
                     .image("busybox")
-                    .cmd("sh", "-c", "md5sum", "<<<", quotedMessage)
+                    .cmd("sh", "-c", "echo $quotedMessage | md5sum")
                     .build()
 
             val creation = docker.createContainer(containerConfig)
             containerId = creation.id()
             docker.startContainer(containerId)
```

Comment:
Please access member variables either using `this.` or by prefixing them with `my`: `myDocker`. The point is that one needs to be able to distinguish between local vars or functions args and memebr variables without IDE assistance, just looking at the text. 

This applies to all codez in this pull request

Reply:
Done